client/src/modules/reports/generate/balance_report/balance_report.html

```diff
@@ -0,0 +1,43 @@
+<bh-report-preview
+  ng-if="ReportConfigCtrl.previewGenerated"
+  source-document="ReportConfigCtrl.previewResult"
+  on-clear-callback="ReportConfigCtrl.clearPreview()"
+  on-save-callback="ReportConfigCtrl.requestSaveAs()">
+</bh-report-preview>
+
+<div ng-show="!ReportConfigCtrl.previewGenerated">
+  <div class="row">
+    <div class="col-md-12">
+      <h3 class="text-capitalize" translate>REPORT.BALANCE_REPORT.TITLE</h3>
+      <p class="text-info" translate>REPORT.BALANCE_REPORT.DESCRIPTION</p>
+    </div>
+  </div>
+
+  <div class="row" style="margin-top : 10px">
+    <div class="col-md-6">
+      <div class="panel panel-default">
+        <div class="panel-heading">
+          <span translate>REPORT.UTIL.OPTIONS</span>
+        </div>
+
+        <div class="panel-body">
+
+        <form name="ConfigForm" bh-submit="ReportConfigCtrl.preview(ConfigForm)" novalidate autocomplete="off">
+
+          <!--select the period by a date-->
+          <bh-date-editor
```

Comment:
I feel like a period selection ("today", "this week", "this month", "this year") would be useful as well to quickly set the date to the beginning of this month (for example).

Additionally, this allows you to set future dates - I'm not sure if you wanted that.  If not, use the `maxDate` directive.